src/qcodes/instrument_drivers/rohde_schwarz/SGS100A.py

```diff
@@ -164,6 +164,8 @@ def __init__(
             set_cmd="SOUR:IQ:IMP:LEAK:I {:.2f}",
             get_parser=float,
             vals=vals.Numbers(-10, 10),
+            unit='%',
+            docstring='Carrier offset for I channel in percentage of the peak envelope power.'
```

Comment:
Could you add the docstring to the line below, so it also shows up in the sphinx docs? Same for the 2 below

Reply:
For reference: documentation is rendered here: https://microsoft.github.io/Qcodes/drivers_api/RohdeSchwarz.html#qcodes.instrument_drivers.rohde_schwarz.RohdeSchwarzSGS100A